resources/hg-CorrespondenceItem.xml

```diff
@@ -0,0 +1,130 @@
+      <path value="Communication.identifier" />
+      <short value="IdentificationNumber" />
+      <definition value="Globally unique number that identifies the instantiation of the information model. The number is composed of an identification of the issuer organization and a unique number assigned by this organization." />
+      <alias value="Identificatienummer" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-1282" />
+        <comment value="IdentificationNumber" />
+      </mapping>
+    </element>
+    <element id="Communication.basedOn">
+      <path value="Communication.basedOn" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-1288" />
+        <comment value="EpisodeOfCare" />
+      </mapping>
+    </element>
+    <element id="Communication.category">
+      <path value="Communication.category" />
+      <binding>
+        <strength value="required" />
+        <description value="valueCodes for general categories of communications such as alerts, instructions, etc." />
+        <valueSet value="https://decor.nictiz.nl/ad/#/hg-/terminology/valueset/2.16.840.1.113883.2.4.3.11.60.103.11.24" />
+      </binding>
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-587" />
+        <comment value="CorrespondenceCategory" />
+      </mapping>
+    </element>
+    <element id="Communication.encounter">
+      <path value="Communication.encounter" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-1285" />
+        <comment value="RelationContact" />
+      </mapping>
+    </element>
+    <element id="Communication.recipient">
+      <path value="Communication.recipient" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-536" />
+        <comment value="CorrespondenceAddressee" />
+      </mapping>
+    </element>
+    <element id="Communication.sender">
+      <path value="Communication.sender" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-382" />
+        <comment value="CorrespondenceSender" />
+      </mapping>
+    </element>
+    <element id="Communication.reasonReference">
+      <path value="Communication.reasonReference" />
+      <type>
+        <code value="Reference" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/hg-Document" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Condition" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Observation" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/DiagnosticReport" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/DocumentReference" />
+      </type>
+      <mapping>
```

Comment:
Is reasonReference the correct element or should it be payload.contentReference?